src/app/components/pagination/pagination.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="pagination" *ngIf="total && current">
+  <div
+    *ngFor="let number of total | pagination : current"
+    class="pagination--unit"
+    (click)="number === undefined || pageSelect.emit(number)"
```

Comment:
It's better to use component function for that logic. Even with simple emitter. Because we try to avoid logic inside the template

Reply:
Moved logic to method.